Caboodle.ruleset

```diff
@@ -0,0 +1,210 @@
+    <Rule Id="SA1129" Action="Warning" />
+    <Rule Id="SA1130" Action="Warning" />
+    <Rule Id="SA1131" Action="Warning" />
+    <Rule Id="SA1132" Action="Warning" />
+    <Rule Id="SA1133" Action="Warning" />
+    <Rule Id="SA1134" Action="Warning" />
+    <Rule Id="SA1135" Action="Warning" />
+    <Rule Id="SA1136" Action="Warning" />
+    <Rule Id="SA1137" Action="Warning" />
+    <Rule Id="SA1139" Action="Warning" />
+    <Rule Id="SA1200" Action="Warning" />
+    <Rule Id="SA1201" Action="Warning" /><!-- TODO: verify SA1201ElementsMustAppearInTheCorrectOrder -->
+    <Rule Id="SA1202" Action="Warning" /><!-- TODO: verify SA1202ElementsMustBeOrderedByAccess -->
+    <Rule Id="SA1203" Action="Warning" />
+    <Rule Id="SA1204" Action="Warning" />
+    <Rule Id="SA1205" Action="Warning" />
+    <Rule Id="SA1206" Action="Warning" />
+    <Rule Id="SA1207" Action="Warning" />
+    <Rule Id="SA1208" Action="Warning" />
+    <Rule Id="SA1209" Action="Warning" />
+    <Rule Id="SA1210" Action="Warning" />
+    <Rule Id="SA1211" Action="Warning" />
+    <Rule Id="SA1212" Action="Warning" />
+    <Rule Id="SA1213" Action="Warning" />
+    <Rule Id="SA1214" Action="Warning" />
+    <Rule Id="SA1215" Action="Warning" />
+    <Rule Id="SA1216" Action="Warning" />
+    <Rule Id="SA1217" Action="Warning" />
+    <Rule Id="SA1300" Action="Warning" />
+    <Rule Id="SA1301" Action="Warning" />
+    <Rule Id="SA1302" Action="Warning" />
+    <Rule Id="SA1303" Action="Warning" />
+    <Rule Id="SA1304" Action="Warning" />
+    <Rule Id="SA1305" Action="None" />
+    <Rule Id="SA1306" Action="Warning" />
+    <Rule Id="SA1307" Action="Warning" />
+    <Rule Id="SA1308" Action="Warning" />
+    <Rule Id="SA1309" Action="Warning" />
+    <Rule Id="SA1310" Action="Warning" />
+    <Rule Id="SA1311" Action="None" />
+    <Rule Id="SA1312" Action="Warning" />
+    <Rule Id="SA1313" Action="Warning" />
+    <Rule Id="SA1314" Action="Warning" />
+    <Rule Id="SA1400" Action="None" />
+    <Rule Id="SA1401" Action="Warning" />
+    <Rule Id="SA1402" Action="Warning" />
+    <Rule Id="SA1403" Action="Warning" />
+    <Rule Id="SA1404" Action="Warning" />
+    <Rule Id="SA1405" Action="Warning" />
+    <Rule Id="SA1406" Action="Warning" />
+    <Rule Id="SA1407" Action="Warning" />
+    <Rule Id="SA1408" Action="Warning" />
+    <Rule Id="SA1409" Action="Warning" />
+    <Rule Id="SA1410" Action="Warning" />
+    <Rule Id="SA1411" Action="Warning" />
+    <Rule Id="SA1412" Action="Warning" />
+    <Rule Id="SA1413" Action="None" />
+    <Rule Id="SA1500" Action="Warning" />
+    <Rule Id="SA1501" Action="Warning" />
+    <Rule Id="SA1502" Action="Warning" />
+    <Rule Id="SA1503" Action="None" />
+    <Rule Id="SA1504" Action="Warning" />
+    <Rule Id="SA1505" Action="Warning" />
+    <Rule Id="SA1506" Action="Warning" />
+    <Rule Id="SA1507" Action="Warning" /><!-- TODO: verify SA1507CodeMustNotContainMultipleBlankLinesInARow -->
```

Comment:
Do we care about multiple lines of open-ness?

Reply:
probably not